charts/steward/templates/clusterrole-run-controller.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-run-controller
```

Comment:
It also needs CRUD for `resourcequotas` and `limitranges` (both in core API group).